src/components/TabNavigation.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Tabs } from '@patternfly/react-core/dist/js/components/Tabs/Tabs';
```

Comment:
Just a note here. It should be safe using esm PF components. They do transform imports from CJS to ESM. We can finally reap all the benefits of ESM build without having duplicate components.

Reply:
So should I run find and replace all? :D

Reply:
It "should" :tm: work. You might even don't have to use absolute imports anymore. But I would definitely check the bundle size after that.